### DISCUSSION TITLE
Issues configuring tiling drag

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#6000] [Q&A] [answered]
I Have been trying to turn off tiling drag by putting the "tiling_drag off" in my i3 config file as stated in the user manual, but it doesnt seem to work even after i3 restarts and system reboots. Does anyone have a clue why this config wont go through?

**Comment by orestisfl** [comment]
Are you trying to disable dragging tiling windows? This setting only
applies to tiling not floating windows.

**Reply by ghost** [reply]
no sir, im trying to disable the function that lets me press my modkey and right click drag to resize my tiling windows. and according to the users manual by i3 that should be possible to disable by adding this "tiling_drag off" to the i3 config file

**Reply by orestisfl** [reply, marked as answer]
No, this is a different function. You can bind the right button to `nop` to disable resizing.

**Reply by ghost** [reply]
my mistake, i completely misunderstand the text i guess. thanks for the fix
